src/RecordManager/Finna/Record/Lido.php

```diff
@@ -211,6 +211,10 @@ public function toSolrArray(\RecordManager\Base\Database\Database $db = null)
         $data['hierarchy_parent_title']
             = $this->getRelatedWorks($this->relatedWorkRelationTypesExtended);
 
+        if ($otherId = $this->getOtherIdentifier()) {
```

Comment:
Mä viittasin sillä getOtherIdentifier**s**-ehdotuksella siihen, että palauttaisi arrayn. Sen voi suoraan tunkea ctrlnum-kenttään sen enempää tarkistelematta. Ja sitten siihen voi lisätä muita tyyppejä tarpeen mukaan, tai laittaa se indeksoimaan kaikki tyypit suoraan (kunhan type-attribuutti on määritelty).

Reply:
Aivan, parempi näin. Muutin indeksoimaan kaikki typet.